app/settings.py

```diff
@@ -66,7 +66,24 @@
 OSF_BASE_URL = env.OSF_BASE_URL.rstrip("/")
 OSF_API_BASE_URL = env.OSF_API_BASE_URL.rstrip("/")
 ALLOWED_RESOURCE_URI_PREFIXES = {OSF_BASE_URL}
-SESSION_COOKIE_DOMAIN = env.SESSION_COOKIE_DOMAIN
+# SESSION_COOKIE_DOMAIN = env.SESSION_COOKIE_DOMAIN
+SESSION_ENGINE = "django.contrib.sessions.backends.cache"
+SESSION_COOKIE_NAME = env.OSF_AUTH_COOKIE_NAME
+OSF_AUTH_COOKIE_SECRET = env.OSF_AUTH_COOKIE_SECRET
+REDIS_HOST = env.REDIS_HOST
+
+CACHES = {
+    "default": {
+        "BACKEND": "django.core.cache.backends.redis.RedisCache",
+        "LOCATION": REDIS_HOST,
+        # 'OPTIONS': {
+        #     'CLIENT_CLASS': 'django_redis.client.DefaultClient',
+        #     'CONNECTION_POOL_KWARGS': {
+        #         'max_connections': 100,
+        #     },
+        # },
```

Comment:
Did we comment this out because this configuration doesn't work with `django.core.cache.backends.redis.RedisCache`? If so, we should just remove the code.